### ISSUE TITLE
Question about S3DIS

### ISSUE THREAD

**Issue by haoranD** [hrzhou2/AdaptConv-master#5]
Dear Haoran,

Thanks for such excellent work.

I got some questions about S3DIS semantic segmentation task.

1. How many epochs do we need to train the model for the S3DIS semantic segmentation task?
2. Can you please give me some suggestions for the model training on S3DIS?
3. If it is possible that you share the trained model for S3DIS semantic segmentation task?

I would be very grateful for your help and hope you have a nice day.

Best Wishes,
Haoran

**Comment by hrzhou2** [comment]
1. The provided train.py is ok with that. It can achieve a good result within 200 epochs.
2. I don't know what is your concern, but if you are changing the batch_num, then absolutely you may also change the max_epochs and learning rate accordingly.
3. Yes

**Comment by haoranD** [comment]
Many thanks for your reply, it is helpful.
